auth_track_and_prevent_brut_force/README.rst

```diff
@@ -0,0 +1,99 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+==============================================
+Authentication Tracking and Prevent Brut Force
+==============================================
+
+This module register each request done by users trying to authenticate into
+Odoo. If the authentication fails, a counter is increased for the given remote
+IP. After 10 attempts (or other quantity, depending on an ir.config_parameter),
+Odoo will ban the remote IP and ignore new requests.
+
+Admin user have the possibility to unblock a banned IP.
+
+This module realizes a call to a web API (http://ip-api.com) to try to have
```

Comment:
Does that call delay the login? It should be then put on Known issues, or prepare an asynchronous call for that